com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -4,6 +4,12 @@
 
 namespace UnityEngine.Rendering.HighDefinition
 {
+    internal enum FPTLMaxLightSizes
+    {
+        Small = 31,
+        High = 63
+    }
```

Comment:
Can't the be declared in the HDRP package to avoid users changing these values ?
Accordingly the comments lines 47 don't have to mention the values, only the enum "text".

Also, nitpicking about the semantics, but if one is "Small", shouldn't the other be "Big" ? Or "Low" / "High" ?
We are referring to a "count" here, so maybe the enum name is also not best chosen, something like "FPTLMaxLightBudget" would be a better fit ?

Reply:
Changed it to low as recommended.